cmd/podman/logs.go

```diff
@@ -104,7 +105,24 @@ func logsCmd(c *cli.Context) error {
 		return err
 	}
 
-	file, err := os.Open(ctr.LogPath())
+	logPath := ctr.LogPath()
+
+	state, err := ctr.State()
+	if err != nil {
+		return err
+	}
+
+	// If the log file does not exist yet and the container is in the
+	// Configured state, it has never been started before and no logs exist
+	// Exit cleanly in this case
+	if _, err := os.Stat(logPath); err != nil {
+		if state == libpod.ContainerStateConfigured {
+			logrus.Debugf("Container has not been created, no logs exist yet")
```

Comment:
Day late dollar short, but shouldn't this read 'Container has not been **started**` ?

Reply:
It's technically correct, as the error will only show up when the container hasn't moved to `ContainerStateCreated`, but agree that your phrasing would make more sense from a user's perspective